src/components/overview/Overview.jsx

```diff
@@ -3,49 +3,112 @@ import App from '../../App.jsx';
 import axios from 'axios';
 import API_KEY from '../../config.js';
 import ImageGallery from './ImageGallery/ImageGallery.jsx';
+import ProductInformation from './ProductInformation/ProductInformation.jsx';
+import ProductDesc from './ProductInformation/ProductDesc.jsx';
+import SocialMedia from './ProductInformation/SocialMedia.jsx';
 
 const Overview = ({productId}) => {
   const [ selectedProductId, updateProductId ] = useState(productId);
   const [ state, updateState ] = useState({
-    productDetailById: {},
-    productStyleById: {},
+    selectedProductDetail: {},
+    selectedProductStyle: {},
+    selectedProductCategory: '',
+    selectedProductDefaultPrice: '',
+    selectedProductDesc: '',
+    selectedProductName: '',
+    selectedProductSlogan: '',
+    selectedProductStyle: {},
+    selectedProductDefaultStyle: {},
```

Comment:
We should probably try to keep the size of the state down... maybe store all this into an object called selectedProduct that then has a bunch of properties?

Reply:
True.. I created some states that I'm not quite using now. But I will definitely make the size down after I start implement the style selector and AddToCart section today. Thanks for pointing this out!

Reply:
Offering a different opinion here: do we really need to worry about the size of state? What's the difference between this and storing all these items as properties in an object- Is it in regards to how props would be passed around? 

I remember, in one of our projects during review week, talking to staff about the size of state. I was also trying to "keep state size down" but state is designed to and optimized to handle a lot of information. In big apps, there will be a lot of items in state. It's one thing if there are items in state that aren't used, but I don't think we should worry too much about the size of state if it's all being used. Perhaps in a later refactor, but for now I think this is fine